public/locales/en-GB/app.json

```diff
@@ -133,6 +134,7 @@
   "Walkie-talkie call": "Walkie-talkie call",
   "Walkie-talkie call name": "Walkie-talkie call name",
   "WebRTC is not supported or is being blocked in this browser.": "WebRTC is not supported or is being blocked in this browser.",
+  "Whether to enable the keyboard shortcuts to e.g. mute/unmute the local microphone.": "Whether to enable the keyboard shortcuts to e.g. mute/unmute the local microphone.",
```

Comment:
I'm not particularly happy with this description, but put it up as a straw man for people to comment on.

Reply:
We could also just explain by example here, like:

"e.g. 'm' to mute/unmute the mic"